package.json

```diff
@@ -1,11 +1,11 @@
 {
   "name": "open-msupply",
   "//": "Main version for the app, should be in semantic version format (any release candidate or test build should be separated by '-' i.e. 1.1.1-rc1 or 1.1.1-test",
-  "version": "1.2.01",
+  "version": "1.3.00",
```

Comment:
Or are we keep going on 1.2.01?

Reply:
I thought 1.3.00 - we had actually discussed and decided to do that for the release just gone and I forgot at the time of release. Personally I like the idea of bumping the minor version with the prescription feature release anyway, so my forgetfulness has paid off!